packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.tsx

```diff
@@ -38,27 +46,22 @@ export const PageRecoveryKeyCreate = (props: RouteComponentProps) => {
     'Back to settings'
   );
 
+  // TODO: Remove feature flag param in FXA-7419
   const navigateBackward = () => {
-    navigate('/settings');
+    navigate(HomePath);
   };
 
   const navigateForward = (e?: React.MouseEvent<HTMLElement>) => {
     e?.preventDefault();
     if (currentStep + 1 <= numberOfSteps) {
       setCurrentStep(currentStep + 1);
     } else {
-      navigate('/settings');
+      // TODO: Remove feature flag param in FXA-7419
+      navigate(HomePath);
     }
   };
 
-  // Redirects to settings if a recovery key is already set
-  // TODO in FXA-7239 - review where user should be redirected if they already have a key created
-  // handle issue where user might get bounced out of the flow if the page is refreshed
-  useEffect(() => {
-    if (account.recoveryKey && !formattedRecoveryKey) {
-      navigate(HomePath, { replace: true });
-    }
-  }, [account, formattedRecoveryKey, navigate]);
+  // TODO prevent page refresh? currently refreshing breaks user from flow
```

Comment:
I think this might have been left over from a previous PR merge (I remember removing this comment at some point but looks like it resurrected) - I'd discussed with Lauren and we weren't happy with the possible solutions (e.g., storing the current step in local storage), and decided it might not be worth the time investment to fix since the current flow has the same behaviour.